packages/amplify-migration-tests/src/__tests__/migration_tests/api.key.migration.test.ts

```diff
@@ -0,0 +1,91 @@
+import { initJSProjectWithProfile, deleteProject, amplifyPush, amplifyPushUpdate } from '../../../../amplify-e2e-tests/src/init';
```

Comment:
Please avoid using relative path from a different package.

Reply:
Including in package.json instead